core/src/main/scala/org/genivi/sota/core/DeviceUpdatesResource.scala

```diff
@@ -84,6 +84,7 @@ class DeviceUpdatesResource(db: Database,
     entity(as[List[PackageId]]) { ids =>
       val f = DeviceUpdates
         .update(id, ids, resolverClient)
+        .flatMap(_ => deviceRegistry.updateLastSeen(id))
```

Comment:
would probably make sense to recover on the future returned by `updateLastSeen`, if that fails we can still respond OK to the client.